exchanges/bitmex/bitmex_wrapper.go

```diff
@@ -558,10 +558,9 @@ func (e *Exchange) GetHistoricTrades(ctx context.Context, p currency.Pair, asset
 	if err != nil {
 		return nil, err
 	}
-	limit := 1000
 	req := &GenericRequestParams{
 		Symbol:  p.String(),
-		Count:   int32(limit),
+		Count:   int32(1000),
```

Comment:
Can make this a const countLimit = 1000 since it's used as a magic number in 4 different areas